docs/src/api/network_analysis_api.md

```diff
@@ -0,0 +1,41 @@
+# [Network analysis and representations](@id api_network_analysis)
+```@meta
+CurrentModule = Catalyst
+```
+
+Note, currently API functions for network analysis and conservation law analysis
+do not work with constant species (currently only generated by SBMLToolkit).
```

Comment:
We allow the definition of constant species in Catalyst (https://docs.sciml.ai/Catalyst/stable/model_creation/dsl_advanced/#dsl_advanced_options_constant_species)?

Also, maybe say SBML generally (I think currently SBMLImporters is the more reliable SBML importer. I think i both covers more cases and also handles jump systems a lot better)